discovery_engine/lib/src/domain/models/source.dart

```diff
@@ -36,6 +36,8 @@ class Source extends Equatable {
   @override
   String toString() => _repr;
 
+  String get value => _repr;
```

Comment:
We could also just make the `_repr` member public (maybe rename it also to `value`), but this is also fine